containers/src/Kokkos_DualView.hpp

```diff
@@ -292,6 +292,15 @@ class DualView : public ViewTraits<DataType, Properties...> {
         d_view(src.d_view),
         h_view(src.h_view) {}
 
+  //! Copy assignment operator (shallow copy assignment)
+  template <typename DT, typename... DP>
+  DualView& operator=(const DualView<DT, DP...>& src) {
+    modified_flags = src.modified_flags;
+    d_view         = src.d_view;
+    h_view         = src.h_view;
+    return *this;
+  }
```

Comment:
Sorry I am a bit late to the party, just going through the changelog to prepare the release 4.2 briefing, and I don't think this is needed.  Converting assignment will just work and do the same thing without us defining it.  It just goes through the converting constructor and regular copy assignment.
Am I missing anything?

https://godbolt.org/z/zehfP4brM

Reply:
It seems it's not needed indeed.

I think I had looked at the case of the view when I made this PR. The view has both the converting copy constructor and the converting copy assignment operator. But I now think I understand that the case if the view is different because it needs to be able to assign unmanaged from managed.

Reply:
I proposed getting rid of it in #6649